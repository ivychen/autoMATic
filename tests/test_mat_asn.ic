// Test matrix element assignment
int main() {
    int matrix[3][3] M;
		int c;

		c = 5;
		M = [[0,0,0]; [0,0,0]; [0,0,0]];
		M[2][2] = c;

	  print(M[2][2]); // print 5
}
