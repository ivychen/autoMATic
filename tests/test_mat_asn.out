
void asnMat(M)
{
int c;
c = 5;
M[2][2] = c;
}
